### PR TITLE
feat: Add Default post language configuration option

### PR DESCRIPTION
**Pull request by last-partizan**
This PR adds option to specify posting language.

Currently, this is exposed only in settings. Next step is to add component to compose form (probably it's best left for another PR, if someone can suggest what widgets i can use for this).

This is my first time working with React, please review it carefully.

Refs #171

### CODE REVIEW

File: app/soapbox/features/preferences/index.tsx [hunk 1]
```diff
@@ -75,6 +75,10 @@ const languages = {
   'zh-TW': '繁體中文（臺灣）',
 };
 
+// This is temporary, need to use full range of ISO languages like here.
+// https://akkoma.dev/AkkomaGang/akkoma-fe/issues/386
+const postLanguages = languages;
```
Comment (last-partizan) [inline app/soapbox/features/preferences/index.tsx:80]: This is temporary, just to see if it works. (And it works).

Akkoma uses iso-639-1 package for this. I can use it too, is that okay?

**Comment by Cl0v1s** [comment]
Thanks a lot for your work there ! I'm working on my side on a language selector for the compose UI so both of those PR are complementary, that's good news ! 

I have a question noneless for a better understanding: why do you add a `language` and `default_language` in app/soapbox/reducers/compose.ts ?

**Comment by last-partizan** [comment]
Because other fields have similar "value" and "default_value" variables. (I think, we can use to check if value has changed, but right now it's unused).

But i may not be necessary. Should i leave it, or maybe better remove now, and add only when it's necessary?

**Comment by Cl0v1s** [comment]
No that's fine, let's keep it for now !

**Pull request merged**